picfixPy/vibrance.py

```diff
@@ -1,20 +1,54 @@
-def vibrance(input_img, intensity=5, output_img):
-    
+def vibrance(input_img, intensity=0, display = False, output = ''):
```

Comment:
Good idea for thinking of the `display` argument, I will add this argument to my function as well.
One thing I'll like to point out here is try not to change the names of the other arguments at this stage. Unless you think our old argument names are horrid, changing the argument names of your function at this stage means you have to change the argument names throughout the entire test suite for this function! Just a thought. I recommend just keep the old argument names for consistency of other functions sake.
```suggestion
def vibrance(input_img, intensity=0, display = False, output_img'):
```

Reply:
I think we also need to add the dependencies of the function on top of the function as well? Unless you don't even have dependencies then very nice!

Reply:
I dont think I understand what you mean by change the names of other arguments, what arguments are changed?

Reply:
oh I am just referring to the argument `output_img` being changed to `output ` here.

Reply:
ah, I'm ok with either, easy fix!